### PR TITLE
[5.1] Adding the missing seeOptionIsSelected and seeCheckboxIsChecked methods

### PR DESCRIPTION
**Pull request by sileence**
These methods will allow developers to test selects, radio groups and checkboxes, along with seeInField that allow developers to test input texts and textareas.

### CODE REVIEW

**Comment by sileence** [comment]
I also added some integration tests this time:

https://github.com/sileence/framework/blob/check_field_values/tests/Foundation/FoundationCrawlerTraitIntegrationTest.php

**Comment by taylorotwell** [comment]
Is there a strong reason not to just name them `seeIsSelected` and `seeIsChecked`?

**Comment by sileence** [comment]
@taylorotwell I just decided to maintain the Codeception name convention. But if you consider the shorter names seeIsSelected and seeIsChecked also makes sense, just let me know, either way is fine.

**Comment by taylorotwell** [comment]
To me the shorter names are better.

**Comment by sileence** [comment]
@taylorotwell how would you abbreviate getSelectedValueFromSelect and getCheckedValueFromRadioGroup? Those are internal methods used by getSelectedValue, and they need to be specific.

**Comment by sileence** [comment]
@taylorotwell done, I changed the names to seeIsSelected and seeIsChecked, I also renamed isCheckboxChecked to isChecked (I'm trying to separate the actual tests (methods prefixed by see*) from the helper methods that interact with the Crawler.

File: src/Illuminate/Foundation/Testing/CrawlerTrait.php [hunk 1]
```diff
@@ -334,7 +367,101 @@ protected function getInputOrTextAreaValue($selector)
...
+            }
+        }
+
+        return;
+    }
+
+    /**
+     * Get the checked value from a radio group.
+     *
+     * @param  Crawler  $radioGroup
```
Comment (GrahamCampbell) [inline src/Illuminate/Foundation/Testing/CrawlerTrait.php]: use FQCN

**Comment by sileence** [comment]
@GrahamCampbell fixed, comments now have FQCNs

**Pull request merged**